src/pages/api/salesConfirmation.ts

```diff
@@ -32,6 +35,16 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
             throw new Error('No fare type session attribute found.');
         }
 
+        const productDating = getSessionAttribute(req, PRODUCT_DATE_ATTRIBUTE);
+        if (!productDating) {
```

Comment:
assume this is being set here rather than the productDateInformation api file. How comes we made the change here. Thought it would make sense in the other file as this is where the logic is. Happy to chat about this

Reply:
Because if we set it there, the confirmation page doesnt know if the user set the value or we did, and we need to know that so we can render (default) or not.